test/runtests.jl

```diff
@@ -18,6 +18,9 @@ end
     @testset "empty row" begin
         include("lp/empty_row.jl")
     end
+    @testset "ensure integer bounds" begin
+        include("lp/ensure_integer_bounds.jl")
+    end
```

Comment:
Same here, MIP-specific reductions should have their own directory and a separate testset.